restfull-api/models/todo.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const todoSchema = new mongoose.Schema({
+  todoText: {
+    type: String,
+    required: true,
+  },
+  todoDate: {
```

Comment:
```suggestion
  createdAt: {
```

Reply:
Let's store the following timestamps also:
* updatedAt
* deletedAt

Reply:
done